wgpu-core/src/device/mod.rs

```diff
@@ -1939,6 +1939,24 @@ impl<B: GfxBackend> Device<B> {
                         },
                     );
                 }
+
+                if let wgt::VertexFormat::Double
+                | wgt::VertexFormat::Double2
+                | wgt::VertexFormat::Double3
+                | wgt::VertexFormat::Double4 = attribute.format
+                {
+                    if !self.features.contains(wgt::Features::VERTEX_ATTRIBUTE_64BIT) {
+                        return Err(pipeline::CreateRenderPipelineError::MissingFeature(
+                            wgt::Features::VERTEX_ATTRIBUTE_64BIT,
+                        ));
+                    }
+                    if !self.features.contains(wgt::Features::SHADER_FLOAT64) {
```

Comment:
not necessary, the `VERTEX_ATTRIBUTE_64BIT` feature should require this one, which is to be checked at device creation, not here